src/Tmds.Ssh/SftpException.cs

```diff
@@ -0,0 +1,20 @@
+// This file is part of Tmds.Ssh which is released under MIT.
+// See file LICENSE for full license details.
+
+using System;
+
+namespace Tmds.Ssh
+{
+    [Serializable]
+    public class SftpException : Exception
+    {
+        public SftpErrorCode ErrorCode { get; internal set; }
+		
+        internal SftpException(string message) : base(message) { }
+        internal SftpException(string message, SftpErrorCode errorCode) : base(message)
```

Comment:
Can you make this `(SftpErrorCode errorCode, string? message)` and handle `null` message here?
You can remove the `SftpException(string message)` constructor.